docs/getting-started/github-apps/github-apps.rst

```diff
@@ -66,6 +66,29 @@ to associate with a workflow on Dockstore. A template for both workflows and ser
 along with explanations for each field. For every branch/release on GitHub that has one of these files, a corresponding entry
 will be made on Dockstore.
 
+Error Handling
+----------------------------------
+Since Dockstore relies on GitHub to tell us when changes have been made on GitHub, there are chances that the message gets lost or delayed.
+Typically, Dockstore reacts within seconds of a change being made on GitHub, however service disruptions can delay this to a few minutes.
+If a message were to get lost, unfortunately you will need to push to GitHub again. Currently, there is no way to tell on Dockstore whether
+a GitHub message was delayed or lost. We recommend waiting a few minutes and then trying to push again. This will be changed in the future.
+
+Another error that could occur is that we received the message from GitHub, however the ``/.dockstore.yml`` is invalid. If we cannot read the 
```

Comment:
If this happens, you may see an error in GitHub app logs. I think it depends how bad it is. :) If the file doesn't exist at all (or has a typo in its name), then you won't get anything in the logs. But I believe if there is a .dockstore.yml file, you'll always see something in GitHub app logs.